includes/Core/Assets/Assets.php

```diff
@@ -445,6 +445,16 @@ private function get_assets() {
 					),
 				)
 			),
+			new Script(
+				'googlesitekit-user',
```

Comment:
```suggestion
				'googlesitekit-datastore-user',
```

Reply:
I guess the AC/IB sort of differ here, but I'm inclined to agree with @aaemnnosttv's suggestion. This is part of the datastore so let's scope it.